sdk/crystal/src/volumes.cr

```diff
@@ -171,5 +171,21 @@ module StorageManager
         StorageManager.error_response(response)
       end
     end
+
+    def rename(new_volname : String)
```

Comment:
accept `new_pool_name` and `new_volname`

Reply:
done